app/src/main/java/com/sun/basic_japanese/alphabet/eachpage/AlphabetPageFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.sun.basic_japanese.alphabet.eachpage
+
+import android.os.Bundle
+import android.support.v7.widget.GridLayoutManager
+import android.support.v7.widget.LinearLayoutManager
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import com.sun.basic_japanese.R
+import com.sun.basic_japanese.alphabet.adapter.AlphabetRecyclerAdapter
+import com.sun.basic_japanese.base.BaseFragment
+import com.sun.basic_japanese.constants.BasicJapaneseConstants
+import com.sun.basic_japanese.constants.BasicJapaneseConstants.ALPHABET_COLUMN_NUMBER
+import com.sun.basic_japanese.data.model.Alphabet
+import com.sun.basic_japanese.data.source.local.AppDatabase
+import kotlinx.android.synthetic.main.fragment_page_alphabet.*
+
+class AlphabetPageFragment : BaseFragment(), AlphabetPageContract.View {
+
+    private var pagePresenter: AlphabetPagePresenter? = null
+    private var alphabetType: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (arguments != null) {
+            val bundle = arguments
+            alphabetType = bundle!!.getString(BasicJapaneseConstants.TYPE)
```

Comment:
không dùng !! nhé vì nó sẽ bỏ qua checknull 
Đoạn này nên sử dụng như này 
```suggestion
            arguments?.run{ alphabetType = getString(...)}
```